### PR TITLE
Fix/android liteserv delete

### PR DESCRIPTION
**Pull request by sethrosetter**
#### Fixes #.

- [x] Ran `flake8`

#### Changes proposed in this pull request:

- if apk exists, remove and attempt to install again

### CODE REVIEW

File: keywords/LiteServAndroid.py [hunk 1]
```diff
@@ -58,9 +58,24 @@ def install(self):
...
+        # If that fails, raise an exception
+        max_retries = 1
+        count = 0
+        while True:
+
+            if count > max_retries:
+                raise LiteServError(".apk install failed!")
+
+            output = subprocess.check_output(["adb", "install", apk_path])
+            if "INSTALL_FAILED_ALREADY_EXISTS" in output or "INSTALL_FAILED_UPDATE_INCOMPATIBLE" in output:
```
Comment (raghusarangapani) [inline keywords/LiteServAndroid.py]: This is not part of stdout. It is probably stderr

**Comment by sethrosetter** [comment]
@raghusarangapani Lets wait on this PR, especially since part of it will mask https://github.com/couchbase/couchbase-lite-java-core/issues/1592

File: keywords/LiteServAndroid.py (hunk 1 above)
Comment (sethrosetter) [inline keywords/LiteServAndroid.py]: ADB streams these to standard output.

**Comment by raghusarangapani** [comment]
Looks good

**Pull request merged**